### ISSUE TITLE
Local CLI command "az webapp up" not working for deployment from Azure OpenAI playground 

### ISSUE THREAD

**Issue by daniel-kress** [microsoft/sample-app-aoai-chatGPT#868]
**Describe the bug**
First of all thanks for your great work! 
We initially deployed the chat app via the Azure OpenAI Chat playground, by clicking the "Deploy to" button. We had already an index prepared in Azure Ai Search and attached that.
For further development, we cloned this repo locally, did some minor code changes and wanted to update the existing deployment. For this, we followed the instructions of your readme file: 
We looked up the python runtime stack and SKU from the already deployed web app and inserted them with the remaining info in the provided az commands:
![image](https://github.com/microsoft/sample-app-aoai-chatGPT/assets/111369638/76661790-bbbb-47f2-883e-1b0b0100657b)

_We are new to Azure webapp deployments. Could you please suggest a solution on how to update the deployment with our local code changes?_

**To Reproduce**

Providing the configuration
1. az webapp up --runtime <runtime-stack> --sku <sku> --name <existing-app-name> --resource-group <resource-group-name>

Deploying the local code
2. az webapp config set --startup-file "python3 -m gunicorn app:app" --name <existing-app-name>

We receive the exception that the python 3.11 runtime is not supported:
![image](https://github.com/microsoft/sample-app-aoai-chatGPT/assets/111369638/4a5811ad-ba2f-40d1-96bb-fb3ebce2562f)
The mentioned list only provides python 3.6 to 3.8...

**Expected behavior**
The az webapp deployment procedure for local code changes should work for the Azure OpenAI deployed web app

**Configuration: Please provide the following**
 - Azure OpenAI model name and version (e.g. 'gpt-35-turbo-16k, version 0613')
 - > "gpt-4"
 - Is chat history enabled?
 - > Yes
 - Are you using data? If so, what data source? (e.g. Azure AI Search, Azure CosmosDB Mongo vCore, etc)
 - > Azure Ai Search

Thanks for providing more insight on how to deploy our local code changes.

**Comment by abhahn** [comment]
Hi @daniel-kress , I think based on the error message its possible there may have been a typo in the value of the runtime used in the `az webapp up` command.

Just to clarify, did you use the value `PYTHON:3.11` in your command, or `PYTHON|3.11`, or something else?  The first option (`PYTHON:3.11`) should be correct -- I'm only asking this based on the content of the error message.  :)

If this part was done correctly, could you share the version of the `az` command line you're using?  I'm just wondering if there is also a possibility of an older version being unaware of the available runtimes supported by App Service.

**Comment by daniel-kress** [comment]
Hi @abhahn  ,

thanks for responding to my issue.

I used your first option `PYTHON:3.11` .

The Azure CLI version in my wsl linux installation is:
`{
  "azure-cli": "2.30.0",
  "azure-cli-core": "2.30.0",
  "azure-cli-telemetry": "1.0.6",
  "extensions": {}
}`

Thanks for the hint I updated the Azure CLI to: 
`{
  "azure-cli": "2.61.0",
  "azure-cli-core": "2.61.0",
  "azure-cli-telemetry": "1.1.0",
  "extensions": {}
}`

Now, the deployment procedure starts, but I get an exception:
`[Errno 22] Invalid argument: '...\\sample-app-aoai-chatGPT\\frontend\\node_modules\\@typescript-eslint\\eslint-plugin\\node_modules\\.bin\\semver'`

For reproduction: I pulled your latest changes from main (#874), executed the start.sh script, stopped the serving process and then tried to deploy my local code with `az webapp up`
I have not done any changes to the frontend code. So, I'm wondering if that issue is happening only on my side.

Thanks
